api/envoy/config/accesslog/v2/file.proto

```diff
@@ -20,14 +20,18 @@ message FileAccessLog {
   // A path to a local file to which to write the access log entries.
   string path = 1 [(validate.rules).string = {min_bytes: 1}];
 
-  // Access log format. Envoy supports :ref:`custom access log formats
-  // <config_access_log_format>` as well as a :ref:`default format
-  // <config_access_log_default_format>`.
   oneof access_log_format {
-    // Access log :ref:`format string<config_access_log_format_strings>`
+    // Access log :ref:`format string<config_access_log_format_strings>`.
+    // Envoy supports :ref:`custom access log formats <config_access_log_format>` as well as a
+    // :ref:`default format <config_access_log_default_format>`.
     string format = 2;
 
-    // Access log :ref:`format dictionary<config_access_log_format_dictionaries>`
+    // Access log :ref:`format dictionary<config_access_log_format_dictionaries>`. All values
+    // are rendered as strings.
     google.protobuf.Struct json_format = 3;
+
+    // Access log :ref:`format dictionary<config_access_log_format_dictionaries>`. Values are
+    // rendered as numbers where appropriate.
```

Comment:
qq: do we have any boolean typed formatter? I guess no?

Reply:
It's possible for a FILTER_STATE command to output booleans, but it's based entirely on protobuf libraries, so I didn't add an explicit test for it.

None of the other command emit booleans.

Reply:
I updated the docs, as I realized I'd missed a couple of things that should be pointed out.